pkgs/desktops/gnome/extensions/pano/default.nix

```
@@ -0,0 +1,96 @@
+{ lib
+, stdenv
+, mkYarnModules
+, fetchFromGitHub
+, atk
+, cogl
+, glib
+, gnome
+, gtk3
+, libgda
+, pango
+, nodePackages
+}:
+
+let
+  version = "v3";
+  uuid = "pano@elhan.io";
+in
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-extension-pano";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "oae";
+    repo = "gnome-shell-pano";
+    rev = version;
```

`nix-env` expects versions to start with a number:

```suggestion
    rev = "v${version}";
```

---

fixed